configcatclient/readwritelock.py

```diff
@@ -30,3 +30,75 @@ def acquire_write(self):
 
     def release_write(self):
         self.__exclude.release()
+
+
+class RWLock(object):
```

Comment:
Can you please remove the old implementation and make this the default ReadWriteLock implementation?

Reply:
Yeah I can do that.